sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -139,16 +149,6 @@ private[hive] object SparkSQLCLIDriver extends Logging {
       // Hive 1.2 + not supported in CLI
       throw new RuntimeException("Remote operations not supported")
     }
-    // Respect the configurations set by --hiveconf from the command line
-    // (based on Hive's CliDriver).
-    val hiveConfFromCmd = sessionState.getOverriddenConfigurations.entrySet().asScala
-    val newHiveConf = hiveConfFromCmd.map { kv =>
-      // If the same property is configured by spark.hadoop.xxx, we ignore it and
-      // obey settings from spark properties
-      val k = kv.getKey
-      val v = sys.props.getOrElseUpdate(SPARK_HADOOP_PROP_PREFIX + k, kv.getValue)
```

Comment:
looks like previously we respect the confs in `sys.props` over the `cmdProperties`?

Reply:
the `cmdProperties` is to latter to take effect here, they don't take part in creating `hive.sessionState`, the `newHiveConf` will be set to SQLConf only